man/bootup.xml

```diff
@@ -34,12 +34,12 @@
     <para>The kernel (optionally) mounts an in-memory file system, often generated by
     <citerefentry project='man-pages'><refentrytitle>dracut</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
     which looks for the root file system. Nowadays this is usually implemented as an initramfs — a compressed
-    archive which is extracted when the kernel boots up into a lightweight in-memory file system based on
-    tmpfs, but in the past normal file systems using an in-memory block device (ramdisk) were used, and the
-    name "initrd" is still used to describe both concepts. It's the boot loader or the firmware that loads
-    both the kernel and initrd/initramfs images into memory, but the kernel which interprets it as a file
-    system. <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry> may
-    be used to manage services in the initrd, similarly to the real system.</para>
+    CPIO archive which is extracted when the kernel boots into a lightweight in-memory file system, either
+    tmpfs or ramfs. In the past normal file systems using an in-memory block device (ramdisk) were used, and
+    the name "initrd" is still used to describe both concepts. It's the boot loader or the firmware that
```

Comment:
i think it's OK to explain the name and where it comes from, but the short time where it was a ramfs that was after it was a ramdisk and before it was a tmpfs is something we shouldn't confuse users with.